{"name": "test_fixture_scope_module02", "status": "failed", "statusDetails": {"message": "AssertionError: assert 18 < 10", "trace": "ramdom_int_in_module = 18\n\n    def test_fixture_scope_module02(ramdom_int_in_module):\n>       assert ramdom_int_in_module < 10\nE       assert 18 < 10\n\ntest_02_fixture.py:50: AssertionError"}, "start": 1627036707126, "stop": 1627036707127, "uuid": "2eea8745-4d39-4898-9c64-ba144659666c", "historyId": "91cccf3b33a98cea60aac657bf4dabdb", "testCaseId": "b3f3e40ec5db963c65c18f67c04ed1c0", "fullName": "test_02_fixture#test_fixture_scope_module02", "labels": [{"name": "suite", "value": "test_02_fixture"}, {"name": "host", "value": "JULIZHOU-6JV5P"}, {"name": "thread", "value": "11292-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_02_fixture"}]}